src/latticpp/ckks/bootstrap_params.h

```diff
@@ -0,0 +1,25 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+#pragma once
+
+#include "latticpp/marshal/gohandle.h"
+#include "cgo/bootstrap_params.h"
+
+namespace latticpp {
+
+    enum NamedBootstrappingParams {
+        // Bootstrapping parameters index 0
+        BootstrapParams_Set2,
+        // Bootstrapping parameters index 1
+        BootstrapParams_Set5,
+        // Bootstrapping parameters index 2
+        BootstrapParams_Set7,
+        // Bootstrapping parameters index 3
+        BootstrapParams_Set4
```

Comment:
Why the set number and the index num are different?

Reply:
The names (`Set*`) are from Lattigo, and are ordered in the same way. Presumably, these sets were given in their paper listed in some order, and they only included a subset of the published parameters in their implementation. As to why they reordered them in their code, I couldn't say. But for our enum to match their enum, we have to use the same order (and names, so that the parameter sets don't lose their semantic meaning relative to the paper which describes what is in each parameter set).